src/Database/Statement/StatementDecorator.php

```diff
@@ -354,6 +355,7 @@ public function lastInsertId(?string $table = null, ?string $column = null)
      */
     public function getInnerStatement()
     {
+        /** @psalm-suppress NullableReturnStatement */
```

Comment:
I feel like this would be cleaner if we catch this and throw meaninful exception?

Reply:
I don't know if the property can actually be null in practice. Some of the db layer internals are not easy to understand so I just chose the safe option :).